docs/faq/general/why-cant-i-see-my-organization.md

```diff
@@ -8,7 +8,13 @@ If you don't see your organization when [adding your organization on Codacy](../
 
 1.  Make sure you have access to the organization with the account you're logged in.
 
-1.  **If you're using GitHub Apps,** [install and authorize Codacy on your organization](https://github.com/apps/codacy-production/installations/new).
+1.  **If you're using GitHub:**
+
+    1.  [Install and authorize Codacy on your organization](https://github.com/apps/codacy-production/installations/new).
+
+    1.  Check if you've reached the [maximum number of organization event webhooks](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks){: target="_blank"} and delete any webhooks that are no longer needed.
```

Comment:
Here I would prefer not to mention the current limit of 20 webhooks explicitly, since the number may change in the future.